### ISSUE TITLE
Can't view multiple accounts at once anymore?

### ISSUE THREAD

**Issue by ceptonit** [zedeus/nitter#770]
I was using the "put usernames separated by a comma" trick to browse multiple accounts in 1 timeline.
It has stopped working as of a couple hours ago. 

The trick actually still works partially.
I don't know if Twitter puts accounts with NSFW content in a specific category, but it seems that any account classed as such don't have their content displayed on the timeline anymore if paired with other accounts.
If the accounts are "SFW" accounts, you can stack them with the comma trick no problem, it will work.
However, if you happen to have 2 NSFW accounts in succession in the url, you get the "No items found" page.
As long as there no 2 NSFW account in succession in your url, the timeline will load, but you will only get content from the SFW accounts in your url.

Basically any NSFW account will not display anything in the timeline with the comma trick. If you happen to have a couple SFW accounts in your url, then only these will show content in the timeline.

Examples: 

This will work fine because all these accounts are SFW
[https://nitter.net/github,NASA,OpenSourceOrg](https://nitter.net/github,NASA,OpenSourceOrg)

This works fine too, but the tweets from Playboy are not shown
[https://nitter.net/NASA,Playboy](https://nitter.net/NASA,Playboy)

This doesn't work as both accounts are NSFW
[https://nitter.net/Playboy,Brazzers](https://nitter.net/Playboy,Brazzers)

Twitter changed something is my guess :eyes:

**Comment by LainLayer** [comment]
For example 2, i can see the tweets from playboy on my side.
![image](https://user-images.githubusercontent.com/10339438/212467571-9638da73-023a-4a29-9c92-e78f8834df04.png)

However example 3 does indeed not show anything. I can see both the playboy and brazzers profiles directly. 
[nitter.net/Brazzers,NASA](https://nitter.net/Brazzers,NASA) does not show brazzers tweets

**Comment by zedeus** [comment]
This has always been a problem, there's no way around it afain

**Comment by ceptonit** [comment]
> This has always been a problem, there's no way around it afaik

Well, the https://nitter.net/Playboy,Brazzers example always worked fine up until yesterday.
Matter of fact, I was using that format with 12 NSFW accounts, not just 2.

I mean, if https://nitter.net/NASA,OpenSourceOrg works, why wouldn't https://nitter.net/Playboy,Brazzers ?

**Comment by KaptainMicila** [comment]
I can confirm what @ceptonit said: a new issue indeed has arised with viewing multiple accounts at once, at least 15 hours ago (when I saw the issue for the first time).
I've got multiple big blocks of accounts (25, the max possible before getting blocked) that I use to browse twitter, but having only 2 accounts that are NSFW will break the feed in those blocks.

**Comment by markcellus** [comment]
Have you tried using RSS feeds? This seems like a good use case for it anyway. 

You could add each profile's RSS feed into a list inside of an RSS reader app which could give you the same result: one feed showing multiple timelines. In fact, it might be better since you can easily add and remove profiles from the list.

**Comment by ceptonit** [comment]
> Have you tried using RSS feeds? This seems like a good use case for it anyway.
> 
> You could add each profile's RSS feed into a list inside of an RSS reader app which could give you the same result: one feed showing multiple timelines. In fact, it might be better since you can easily add and remove profiles from the feed.

Yes, RSS is the obvious workaround. But it's definitely not as convenient.
The only advantage of RSS is the no limit on the amount of accounts you can view with 1 URL.

RSS drawbacks:
- Need an app or an extra 3rd party website to handle the feeds.
- You have to handle Read vs Unread posts or else your feeds grow larger and larger. With Nitter, they never go away read or unread at you don't have to worry about storage management.
- A bit problematic with re-tweets or embedded content (videos mostly).
- Everything is displayed as a succession of posts which is less eye pleasing vs the layout of proper tweet display.

**Comment by ceptonit** [comment]
Everything works fine again!

Looks Twitter must have reverted/fixed something :smile: 

Closing this as it's no longer an issue.
